pubspec.lock

```diff
@@ -1,34 +1,55 @@
 # Generated by pub
```

Comment:
The `pubspec.lock` file should be part of `.gitignore` for a library.  See [Flutter .gitignore Template](https://github.com/toptal/gitignore/blob/master/templates/Flutter.gitignore#L16)